cmd/wallet-web/src/components/DIDManagement/ImportDid.vue

```diff
@@ -110,8 +115,16 @@ export default {
     ...mapGetters('agent', { getAgentInstance: 'getInstance' }),
     ...mapGetters(['getCurrentUser']),
     updateAllDIDs: async function () {
+      console.log('this.getCurrentUser().profile.token', this.getCurrentUser().profile.token);
```

Comment:
Do we need these console.log  lines here ?

Reply:
Removed